lib/src/json_editor_main.dart

```diff
@@ -32,40 +34,50 @@ class JsonEditor extends StatefulWidget {
           String? jsonString,
           bool enabled = true,
           bool openDebug = false,
-          ValueChanged<JsonElement>? onValueChanged}) =>
+          ValueChanged<JsonElement>? onValueChanged,
+          ValueChanged<String>? onErrorChanged,
```

Comment:
`ValueChanged<String>? onErrorChanged` to `ValueChanged<Object>? onError` is better.